src/core/Element.ts

```diff
@@ -15,12 +15,12 @@ abstract class Element extends EventEmitter {
   public set timer(timer: NodeJS.Timeout) {}
 
   public onDestroy(listener: (...arg: any[]) => void) {
-    this.on(EVENT.ELEMENT.DESTORY, listener);
```

Comment:
In vscode, can use this extension to check typo:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker